slash-commands/slash-commands.js

```diff
@@ -67,15 +77,19 @@ CandyShop.SlashCommands = (function(self, Candy, $) {
 						var command = match[1];
 						var data = match[2];
 
+						// pass though some commands, they only merit formatting elsewhere
+						if ($.inArray(command, self.passthrough) != -1) { } 
 						// Match only whitelisted commands
-						if ($.inArray(command, self.commands) != -1) {
+						else if ($.inArray(command, self.commands) != -1) {
 							self[command](data);
-						} else {
-							// TODO: Better way to notify the user of the invalid command
-							alert("Invalid command: " + command);
+							args.message = '';
+						}
+						else {
```

Comment:
You should provide a way for the user to type a message starting with a slash, in other clients I’ve seen a /say command used for that, and also a message starting with // which would then be reduced to only one / when sending it.